.circleci/verbatim-sources/nightly-build-smoke-tests-defaults.yml

```diff
@@ -12,6 +12,8 @@ smoke_linux_test: &smoke_linux_test
       at: ~/workspace
   - attach_workspace:
       at: /home/circleci/project
+  - run:
+      <<: *should_run_job
```

Comment:
These will never run on PRs or on master, so these aren't needed either. These are the jobs that run everyday, download pytorch with the official pip/conda instructions, and then run smoke tests on them. They are thus only affected by the binaries of the previous night and never need to be run on master or PRs